src/Reflection/ParametersAcceptorSelector.php

```diff
@@ -203,6 +204,31 @@ public static function selectFromArgs(
 				];
 			}
 
+			if (isset($args[0]) && (bool) $args[0]->getAttribute(ImplodeArgVisitor::ATTRIBUTE_NAME)) {
```

Comment:
I don't think the isset is necessary

```suggestion
			if ((bool) $args[0]->getAttribute(ImplodeArgVisitor::ATTRIBUTE_NAME)) {
```

see also https://github.com/phpstan/phpstan-src/pull/4160